### PR TITLE
feat(HII-5211): Action separates migrations from nodejs startup script

### PR DESCRIPTION
**Pull request by koppathachan**

### CODE REVIEW

File: composite-actions/nodejs-generic-api/migrate/action.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    - uses: extenda/actions/gcp-secret-manager@v0
+      with:
+        service-account-key: ${{ inputs.GCLOUD_AUTH }}
+        secrets: |
+          DB_HOST: postgresql_user_profile_private_address
+          DB_PORT: 5432
+          DB_NAME: postgres_user_profile_db_name
+          DB_USER: postgres_user_profile_role_name
+          DB_PASS: postgres_user_profile_role_password
```
Comment (glebbash) [inline composite-actions/nodejs-generic-api/migrate/action.yaml:46]: This secrets are for UPS db only

File: composite-actions/nodejs-generic-api/migrate/action.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    - name: Install dependencies
+      if: steps.cache-node-modules.outputs.cache-hit != 'true'
+      shell: bash
+      run: npm ci
+
+    - uses: extenda/actions/gcp-secret-manager@v0
+      with:
+        service-account-key: ${{ inputs.GCLOUD_AUTH }}
+        secrets: |
```
Comment (artemka-debug) [inline composite-actions/nodejs-generic-api/migrate/action.yaml:41]: What will happen, if those secrets are not defined in GCP?

**Review by glebbash — CHANGES_REQUESTED** [review]

File: composite-actions/nodejs-generic-api/migrate/action.yaml (hunk 2 above)
Comment (artemka-debug) [reply composite-actions/nodejs-generic-api/migrate/action.yaml:41]: Like in other projects, we have defferent names for secrets

**Review by artemka-debug — CHANGES_REQUESTED** [review]

**Review by glebbash — APPROVED** [review]

**Review by glebbash — CHANGES_REQUESTED** [review]
Please name this action `prisma-migrate` or something more specific then just `migrate`

**Comment by koppathachan** [comment]
> Please name this action `prisma-migrate` or something more specific then just `migrate`

I will rename it to db-migrate since it need not be specific to prisma? @glebbash

**Comment by glebbash** [comment]
Maybe not prisma but CloudSQL instead, the set of envs is very specific and won't work for Spanner for example

**Review by mdovhopo — APPROVED** [review]

**Pull request merged**